locales/ka-GE/src/vision.ts

```diff
@@ -2,67 +2,68 @@ import {removeUndefinedLocaleResources} from 'sanity'
+  'query.label': 'მოძებნა',
   /** Label for the "Query URL" field, shown after executing a query, and allows for copying */
-  'query.url': undefined, // 'Query URL'
+  'query.url': 'მოძებნის URL',
 
   /** Label for "End to End time" information of the fetched query */
-  'result.end-to-end-time-label': undefined, // 'End-to-end'
+  'result.end-to-end-time-label': 'ბოლოდან ბოლომდე',
   /** Label for "Execution time" information of the fetched query */
-  'result.execution-time-label': undefined, // 'Execution'
+  'result.execution-time-label': 'შესრულების დრო',
   /** Label for "Result" explorer/view */
-  'result.label': undefined, // 'Result'
+  'result.label': 'შედეგი',
   /**
    * "Not applicable" message for when there is no Execution time or End to End time information
    * available for the query (eg when the query has not been executed, or errored)
    */
-  'result.timing-not-applicable': undefined, // 'n/a'
+  'result.timing-not-applicable': 'არ არის მიმართულება',
 
   /** Label for the "API version" dropdown in settings */
-  'settings.api-version-label': undefined, // 'API version'
+  'settings.api-version-label': 'API ვერსია',
   /** Label for the "Custom API version" input in settings, shown when "other" is chosen as API version */
-  'settings.custom-api-version-label': undefined, // 'Custom API version'
+  'settings.custom-api-version-label': 'საკუთარი API ვერსია',
   /** Label for the "Dataset" dropdown in vision settings */
-  'settings.dataset-label': undefined, // 'Dataset'
+  'settings.dataset-label': 'Dataset',
   /** Error label for when the API version in 'Custom API version' input is invalid */
-  'settings.error.invalid-api-version': undefined, // 'Invalid API version'
+  'settings.error.invalid-api-version': 'არასწორი API ვერსია',
   /** Label for the "other" versions within the "API version" dropdown */
-  'settings.other-api-version-label': undefined, // 'Other'
+  'settings.other-api-version-label': 'სხვა',
   /**
    * Label for the "Perspective" dropdown in vision settings
    * @see {@link https://www.sanity.io/docs/perspectives}
    */
-  'settings.perspective-label': undefined, // 'Perspective'
+  'settings.perspective-label': 'Perspective',
   /** Call to action to read the docs related to "Perspectives" */
-  'settings.perspectives.action.docs-link': undefined, // 'Read docs'
+  'settings.perspectives.action.docs-link': 'დოკუმენტაციის წაკითხვა',
   /** Description for popover that explains what "Perspectives" are */
-  'settings.perspectives.description': undefined, // 'Perspectives allow your query to run against different "views" of the content in your dataset'
+  'settings.perspectives.description':
+    'Perspectives გაძლევთ შესაძლებლობას გაუშვათ თქვენი მოძებნა სხვადასხვა "ხედვებიდან" თქვენი dataset-ის კონტენტზე',
   /** Title for popover that explains what "Perspectives" are */
-  'settings.perspectives.title': undefined, // 'Perspectives'
+  'settings.perspectives.title': 'Perspectives',
 })
```

Comment:
'action.copy-url-to-clipboard': 'დაკოპირებულია',
'action.listen-execute': 'მოსმენა',
'result.end-to-end-time-label': 'ბო ბოლომდე',
 'result.end-to-end-time-label': 'დასასრულის დრო',